sylt-parser/src/statement.rs

```diff
@@ -243,16 +262,20 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
                     } else {
                         ctx.prev().span()
                     };
-                    let name = PathBuf::from(bare_name)
+                    let name = PathBuf::from(&name)
                         .file_stem()
                         .unwrap()
                         .to_str()
                         .unwrap()
                         .to_string();
-                    (ctx, Identifier { span, name })
+                    (ctx, UseIdentifier::Implicit(Identifier { span, name }))
                 },
             };
-            (ctx, Use { ident, file })
+            let name = Identifier {
+                span: name_span,
+                name,
```

Comment:
Doesn't this name need to keep the slashes at the start and end? I think this might strip them.

Reply:
I think you're right, yes. Will try and maybe fix.